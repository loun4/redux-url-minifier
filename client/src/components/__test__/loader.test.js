import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '../loader';

test('Render loader', () => {
  const component = renderer.create(<Loader />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
