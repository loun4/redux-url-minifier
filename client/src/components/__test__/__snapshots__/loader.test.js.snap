// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render loader 1`] = `
<div
  className="loader"
>
  <img
    alt=""
    height="60"
    src="loader.svg"
    width="60"
  />
</div>
`;
